aws/main.tf

```diff
@@ -1,6 +1,17 @@
+data aws_eks_cluster "my_eks" {
+  count = var.modules_info.eks.create ? 0 : 1
+  name = var.modules_info.eks.cluster_id
+}
+
+locals {
+  vpc_id = (var.modules_info.efs.create && var.modules_info.eks.create && var.modules_info.vpc.create) ? module.vpc[0].vpc_id : var.modules_info.vpc.id
```

Comment:
Why is `var.modules_info.efs.create && var.modules_info.eks.create && var.modules_info.vpc.create` necessary? Isn't `var.modules_info.vpc.create` enough with the variable validations?

Reply:
ye, you are right. 
fixing.